sup3r/models/wind_conditional_moments.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""Wind conditional moment estimator with handling of low and
+high res topography inputs."""
+import logging
+import tensorflow as tf
+
+from sup3r.models.abstract import AbstractWindInterface
+from sup3r.models.conditional_moments import Sup3rCondMom
+
+
+logger = logging.getLogger(__name__)
+
+
+class WindCondMom(Sup3rCondMom, AbstractWindInterface):
```

Comment:
What's the reason for defining tf_generate_wind and then using it to define tf_generate? Same for generate_wind, and set_model_params_wind? I think we could just override the standard names in this class

Reply:
So `Sup3rCondMom` already has a `tf_generate` method that we need to redefine to handle the topography. We have 2 choices, either I redefine `tf.generate` in `WindCondMom` and `WindGan` but this create redundancy since `tf_generate` is the same for both. The alternative is what I did but then I have to refer to this method with a different name, otherwise WindCondMom and WindGan would not know which one to grab.

Reply:
Yeah I missed it in WindGan. This is best option. Just those little things I mentioned in the batch handling and you're good to go!